javascript/packages/orchestrator/src/test-runner/assertions.ts

```diff
@@ -89,6 +89,15 @@ const CalcMetrics = ({
   timeout,
 }: FnArgs) => {
   const comparatorFn = comparators[op!];
+  const mathFn =
+    math_ops === "Minus"
+      ? (a: number, b: number): number => {
+          return a - b;
+        }
+      : (a: number, b: number): number => {
+          return a + b;
+        };
```

Comment:
You could use the math_ops value inside the closure ?

```suggestion
        (a: number, b: number): number => {
          return math_ops === "Minus ? a - b : a + b;
        }
```